index.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable max-classes-per-file */
+
```

Comment:
- [ ] Kindly remove this Eslint disabling comment from this file. `Eslint` helps us to write clean, maintainable and readable codes. Using Eslint disabling comments defeats this purpose. Kindly remove it. 👍